FirebaseDatabase/Tests/Unit/Swift/DatabaseAPITests.swift

```diff
@@ -623,6 +625,10 @@ final class DatabaseAPITests {
       break
     case .value:
       break
+    case .none:
+      break
+    case .some:
+      break
```

Comment:
nit: you can shorten this code by putting value, none, and some in the same statement.

```swift
case .value, .none, .some:
  break
```

Reply:
Just to confirm, `.value` is added by `DataEventType` while `.none` and `.some` aren't. Should all three still be grouped or should I just group `.none` and `.some`?

Reply:
It's up to you :)